flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/api/bridge/java/StreamTableEnvironment.java

```diff
@@ -454,6 +454,12 @@ Table fromChangelogStream(
      * <p>If the input table contains a single rowtime column, it will be propagated into a stream
      * record's timestamp. Watermarks will be propagated as well.
      *
+     * <p>Specifically, if the input table wraps around and describes a relational operation that
```

Comment:
Would it be simpler to say this:

```
 * <p>If the input table is created by {@link #fromDataStream(DataStream)}}, the original
 * DataStream is returned.
 *
 * <p>If the input table contains a single rowtime column, it will be propagated into a stream
 * record's timestamp. Watermarks will be propagated as well.
```